fourmolu.yaml

```diff
@@ -6,6 +6,8 @@ diff-friendly-import-export: false # 'false' uses Ormolu-style lists
 respectful: false # don't be too opinionated about newlines etc.
 haddock-style: single-line # 'signle-line' or 'multi-line', i.e., '--' vs. '{-'
 newlines-between-decls: 1 # number of newlines between top-level declarations
+ghc-opt:
+  -XImportQualifiedPost
```

Comment:
Why do we need this in `fourmolu.yaml` _and_ in `flake.nix`? IIUC, `flake.nix` is going to set it has an environment variable, so having it in both places is redundant? Of course I might just not understand correctly what `flake.nix` is going ^_^;

Reply:
oh that's a mistake! fourmolu actually doesn't look at this field in the config file

Reply:
Filed the bug https://github.com/cachix/pre-commit-hooks.nix/issues/171